2023-02-04 20:37:55,668 INFO    MainThread:13423 [internal.py:wandb_internal():89] W&B internal server running at pid: 13423, started at: 2023-02-04 20:37:55.667520
2023-02-04 20:37:55,675 DEBUG   HandlerThread:13423 [handler.py:handle_request():126] handle_request: run_start
2023-02-04 20:37:55,676 INFO    WriterThread:13423 [datastore.py:open_for_write():77] open: /home/bhavyajeet.singh/msme/MEMS-XF2T/generation/mT5-baseline/wandb/offline-run-20230204_203753-w2270rec/run-w2270rec.wandb
2023-02-04 20:38:00,204 DEBUG   HandlerThread:13423 [meta.py:__init__():40] meta init
2023-02-04 20:38:00,204 DEBUG   HandlerThread:13423 [meta.py:__init__():54] meta init done
2023-02-04 20:38:00,205 DEBUG   HandlerThread:13423 [meta.py:probe():211] probe
2023-02-04 20:38:00,214 DEBUG   HandlerThread:13423 [meta.py:_setup_git():201] setup git
2023-02-04 20:38:00,268 DEBUG   HandlerThread:13423 [meta.py:_setup_git():208] setup git done
2023-02-04 20:38:00,269 DEBUG   HandlerThread:13423 [meta.py:_save_pip():58] save pip
2023-02-04 20:38:00,271 DEBUG   HandlerThread:13423 [meta.py:_save_pip():72] save pip done
2023-02-04 20:38:00,272 DEBUG   HandlerThread:13423 [meta.py:_save_conda():79] save conda
2023-02-04 20:38:13,603 DEBUG   HandlerThread:13423 [meta.py:_save_conda():87] save conda done
2023-02-04 20:38:13,604 DEBUG   HandlerThread:13423 [meta.py:probe():249] probe done
2023-02-04 20:40:08,149 WARNING MainThread:13423 [internal.py:wandb_internal():145] Internal process interrupt: 1
2023-02-04 20:40:08,706 WARNING MainThread:13423 [internal.py:wandb_internal():145] Internal process interrupt: 2
2023-02-04 20:40:08,706 ERROR   MainThread:13423 [internal.py:wandb_internal():148] Internal process interrupted.
2023-02-04 20:40:08,708 INFO    SenderThread:13423 [sender.py:finish():986] shutting down sender
2023-02-04 20:40:09,514 INFO    WriterThread:13423 [datastore.py:close():281] close: /home/bhavyajeet.singh/msme/MEMS-XF2T/generation/mT5-baseline/wandb/offline-run-20230204_203753-w2270rec/run-w2270rec.wandb
2023-02-04 20:40:09,518 INFO    HandlerThread:13423 [handler.py:finish():683] shutting down handler
2023-02-04 20:40:09,522 INFO    MainThread:13423 [internal.py:handle_exit():76] Internal process exited
